### ISSUE TITLE
[Feature Request] Chromecast support

### ISSUE THREAD

**Issue by OJ7** [pulsejet/memories#555]
**Is your feature request related to a problem? Please describe.**
When showing others photos/videos on my phone, it's hard to show it to everyone In the room on a tiny screen. 

**Describe the solution you'd like**
It would be nice to be able to cast specific photos/videos on the TV (via Chromecast) for everyone to see similar to how Google Photos does it. 

**Describe alternatives you've considered**
Mirroring full screen on TV, but that requires showing your whole library on the TV instead of the ones you choose. Also, screen mirroring on Chromecast is only available via Android (and maybe Chrome-based browsers on desktop?), not for iOS devices. 

Sharing a video to the Web Cast Video app kind of works, but it's a little buggy and only supports videos, not photos. 

**Additional context**

**Comment by pulsejet** [comment]
Indeed, this would be very nice, but the chromecast API is weird esp on web. Gonna add this to the milestone for investigation though.

**Comment by pulsejet** [comment]
I investigated this but as I suspected this is quite contrived (Google as usual). Moving back to backlog.

**Comment by mimedo** [comment]
@pulsejet What's the status of this feature request? Chromecast support would be a gamechanger to cast photos and videos to the a cast supported receiver device to share memories with friends and family.
What's the burden to integrate?

**Comment by pulsejet** [comment]
> I investigated this but as I suspected this is quite contrived (Google as usual). Moving back to backlog.

Nothing since here. Don't know if the API improved in any way

**Comment by mimedo** [comment]
What about your roadmap? Any chance to prioritize this feature?
